src/components/product_select/CreditCard.js

```diff
@@ -1,8 +1,8 @@
 import React from 'react';
 
 import ProductCard from './ProductCard.js';
-import cardImage from './images/card.svg';
-import * as colors from '../colors.js';
+import cardImage from '../images/card.svg';
+import * as colors from '../../helpers/colors.js';
```

Comment:
Let's try to change to absolute imports soon. Not in this PR, but I'd love to see it in the next one.